core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -120,6 +120,9 @@ abstract class Chunk[+O] extends Serializable with ChunkPlatform[O] with ChunkRu
     Chunk.array(b.result()).asInstanceOf[Chunk[O]]
   }
 
+  /** Returns a chunk that has only the elements that do not satisfy the supplied predicate. */
+  def filterNot(p: O => Boolean): Chunk[O] = filter(p.andThen(!_))
```

Comment:
Thank you taking on this! A minor note, wouldn't it be simpler and neater?
```suggestion
  def filterNot(p: O => Boolean): Chunk[O] = filter(!p(_))
```